### ISSUE TITLE
Status bar doesn’t blur until you scroll very far

### ISSUE THREAD

**Issue by Jarboer** [himynameisubik/Kalm-support#238]
General information:
===

**Kalm version:**

3.0.0

**Kalm preferences file:**

[https://pastebin.com/GNertT43](https://pastebin.com/GNertT43)

Device information:
===

**iOS version and language:**

iOS 14.2 and English

**Device:**

iPhone 12

**Jailbreak:**

Unc0ver

Bug description:
===

**Describe the bug:**

The status bar text like your carrier doesn't get the normal blurred background like on stock iOS until you scroll down a lot. Normally on stock iOS the status bar becomes blurred as soon as the clock touches the status bar so that it is ￼legible.

**How to reproduce:**

1. Go to your Lock Screen
2. Scroll down a little
3. Notice that the status bar doesn't become blurred

**Expected behavior:**

I'd expect the that status bar would get blurred as soon as it overlaps with the area where the clock is and beyond.

**Screenshots:**

![image](https://user-images.githubusercontent.com/16063018/109911100-7755db80-7c77-11eb-9352-774c8e6d34a6.jpeg)

**Comment by himynameisubik** [comment]
Not sure if this can be done. And it's not really a bug since what you describe is not stock iOS behavior. See the screenshot without Kalm:
![IMG_0107 2](https://user-images.githubusercontent.com/322449/109931686-6d5cc880-7cc9-11eb-9e45-8c483737efc2.PNG)

I'll look into it, but no promise.

**Comment by Jarboer** [comment]
Yea sorry, I messed up and miss remembered as it does this on non-notched devices, since I was comparing to my iPad. On notched devices the time goes right through the notch and so the status bar doesn't interact with other text until it reaches notifications, which is when it starts to blur it. Still would be a nice addition, if you are able to add it. This is what it looks like when it blurs due to the status bar being over notifications: ![image](https://user-images.githubusercontent.com/16063018/110067227-4346ed00-7d41-11eb-82d7-e9166b34a5ab.jpeg)

**Comment by himynameisubik** [comment]
Yeah exactly, the time lines up with the notch. I'll add it to my list, but keep in mind as I mentioned, feature requests are closed and on hold.

**Comment by CySxL** [comment]
@himynameisubik Hi! I've seen the pinned issues about feature request is closed, so I reached you out via Twitter with a I think really small and easy-to-achieve feature. 

Dunno have you seen it already ?

Link here :
https://twitter.com/cysxl/status/1367034384985956354?s=21
